src/ocean/core/SmartUnion.d

```diff
@@ -81,6 +81,15 @@ struct SmartUnion ( U )
         return this._.active_names[this._.active];
     }
 
+    /***************************************************************************
+
+        Helper mixin to support final switch over a smart union: it handles the
+        `case none:` in D2-build and `default:` in D1.
```

Comment:
"and `default:` (including `case none:`) in D1."

Reply:
Updated.